src/core/modes/raindex/index.test.ts

```
@@ -0,0 +1,980 @@
+// mocks
+vi.mock("../../../common", async (importOriginal) => ({
+    ...(await importOriginal()),
+    extendObjectWithHeader: vi.fn(),
+}));
+
+vi.mock("sushi/currency", async (importOriginal) => {
+    return {
+        ...(await importOriginal()),
+        Token: class {
+            constructor(args: any) {
+                return { ...args };
```

Biome (2.3.13)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)